src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@trxcllnt I tried w/ `import * ...` but it didn't work. I am more inclined to think about an issue w/ the DataSource like I said before; w/ my last commit I tried to simplify the context so that you will be able to run and test locally this app:

```bash
yarn bundle
NASA_API_KEY=DEMO_KEY yarn start:dev
```

thanks,